src/components/Table/GridTable.tsx

```diff
@@ -274,6 +290,111 @@ export function GridTable<R extends Kinded, X extends Only<GridTableXss, X> = an
   const expandedColumnIds: string[] = useComputed(() => tableState.expandedColumnIds, [tableState]);
   const columnSizes = useSetupColumnSizes(style, columns, resizeTarget ?? resizeRef, expandedColumnIds);
 
+  function isCursorBelowMidpoint(target: HTMLElement, clientY: number) {
```

Comment:
Minor: Maybe move to the utils file given afaict the function doesn't need to capture any state.